spec/rails_helper.rb

```diff
@@ -20,6 +21,12 @@
   exit 1
 end
 
+Capybara.register_driver :headless_chrome do |app|
+  options = Selenium::WebDriver::Chrome::Options.new
+  options.add_argument('--headless=new')
+  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+end
+
```

Comment:
so we already have cubrite setup `spec/system/support/cuprite_setup.rb`

I happy to switch to selenium but would prefer not to have both.. :)

Reply:
I hadn't even spotted cuprite! It doesn't play nicely with axe, so I've removed it, using selenium's headless chrome as the replacement. It has the advantage that it supports capybara's `drop` so we're a step closer to adding that missing spec in `reason_why_spec.rb` (I started to build it but don't understand enough of the infrastructure so I've left a TODO in place to finish it off).